src/components/IndicatorTypeMark/IndicatorTypeMark.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import getColorVariant from '../../helpers/getColorVariant';
+import style from './IndicatorTypeMark.scss';
+import COLORS from '../../style/_colors.scss';
+
+const classnames = require('classnames/bind').bind(style);
+
+const SIZES = {
+  SMALL: 'SMALL',
+  LARGE: 'LARGE',
+};
+
+// A "P" in a circle
+const IndicatorTypeMark = (props) => {
+  const performanceMarkColor = getColorVariant(props.colorName, '900');
+
+  let pColor;
+  if (props.isCategoryPage) {
+    pColor = COLORS.WHITE;
```

Comment:
I can't see where this gets used....?

Reply:
Wow, nicely caught. I started out just falling back to CSS for color, but got half-way to explicitly setting the colour here for clarity.

Fixed.